### PR TITLE
Update get_description.py script

### PR DESCRIPTION
**Pull request by amnonh**
get_description.py script is a document related script that looks for metrics description in the code.
Its configuration needs to address changes in the code.

This series contains a configuration change and a code fix that allows it to run as a standalone script, and not as a library.

No need to backport, this a documentation related script.

### CODE REVIEW

**Comment by tchaikov** [comment]
> #### 🔴 CI State: FAILURE
> 
> ✅ - [Build](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/10644/) ✅ - [Container Test](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/10644/) ✅ - [dtest with topology changes](https://jenkins.scylladb.com/job/scylla-master/job/gating-dtest-release-with-consistent-topology-changes/2822/) ❌ - [dtest](https://jenkins.scylladb.com/job/scylla-master/job/gating-dtest-release/10843/) ✅ - [Unit Tests](https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/10644/)
> #### Failed Tests (1/31535):
> 
>     * [test_ignore_dead_nodes_for_whole_dc_replace_option](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/10644/testReport/junit/repair_based_node_operations_test/TestRepairBasedNodeOperations/Tests___dtest___test_ignore_dead_nodes_for_whole_dc_replace_option) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+test_ignore_dead_nodes_for_whole_dc_replace_option)
> 
> 
> #### Build Details:
> 
>     * Duration: 2 hr 44 min
> 
>     * Builder: spider1.cloudius-systems.com

#19907

File: scripts/metrics-config.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "100": execution_stages
 "seastar/src/core/fair_queue.cc":
     groups:
-        "400": io_queue
+        "350": io_queue
```
Comment (denesb) [inline scripts/metrics-config.yml]: unrelated change?

File: scripts/metrics-config.yml [hunk 2]
```diff
@@ -62,4 +66,3 @@
         "_queue_name + \"_rx_bytes\"": ["queue"]
         "_queue_name + \"_tx_frags\"": ["queue"]
         "_queue_name + \"_rx_frags\"": ["queue"]
-"seastar/tests/unit/metrics_tester.cc": skip
```
Comment (denesb) [inline scripts/metrics-config.yml]: you seem to undo the first patch here

Comment (amnonh) [reply scripts/metrics-config.yml]: I've just stashed the two changes

**Review by denesb — APPROVED** [review]
